flexget/plugins/estimators/est_release_series_tvmaze.py

```diff
@@ -22,7 +22,7 @@ class EstimatesSeriesTVMaze(object):
     def estimate(self, entry):
         if not all(field in entry for field in ['series_name', 'series_season', 'series_episode']):
             return
-        series_name = entry['series_name']
+        series_name = entry['series_name'].encode('utf-8')
```

Comment:
Hmm, I read this backwards maybe jumped the gun with my comment. I'll keep looking.